### PR TITLE
Add sg commands to build wolfi packages and images locally

### PR DESCRIPTION
**Pull request by willdollman**
As part of improving the Wolfi dev experience, we want to be able to build packages and images locally using sg. It should also be easy for developers to refer to packages they have built locally in their images, for testing. Currently this is difficult due to repo signatures and docker mount points.

https://github.com/sourcegraph/security/issues/563

## Test plan

- Manually test commands
- CI

### CODE REVIEW

File: dev/sg/internal/wolfi/base-image.go [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	err := cmd.Run()
+	if err != nil {
+		return errors.Wrap(err, "failed to build base image")
+	}
+
+	std.Out.Write("")
+	std.Out.WriteLine(output.Linef(output.EmojiSuccess, output.StyleSuccess, "Successfully built base image %s\n", name))
```
Comment (evict) [inline dev/sg/internal/wolfi/base-image.go:71]:
```suggestion
	std.Out.WriteSuccessF("Successfully built base image %s\n", name)
```

File: dev/sg/internal/wolfi/package.go [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+	}
+	if err := os.MkdirAll(c.KeyDir, os.ModePerm); err != nil {
+		return c, err
+	}
+	if err := os.MkdirAll(filepath.Join(c.ImageDir, c.Arch), os.ModePerm); err != nil {
+		return c, err
+	}
+
+	// Generate keys for local repository
+	f, err := os.Open(c.KeyFilepath)
```
Comment (evict) [inline dev/sg/internal/wolfi/package.go:54]: It's an existence check? Why not use `os.Stat`?

File: dev/wolfi/build-local-base-image.sh [hunk 3]
```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+###
+### DEPRECATED in favour of `sg wolfi image`
```
Comment (evict) [inline dev/wolfi/build-local-base-image.sh:6]: Why include it still?

**Review by evict — APPROVED** [review]
Just some nits. There are some instances where you could use `std.Out.WriteSuccessF` instead of using `LineF` with `WriteF`. But looks good!

File: dev/sg/internal/wolfi/base-image.go (hunk 1 above)
Comment (evict) [reply dev/sg/internal/wolfi/base-image.go:71]: You can also do this in the other instances.

Comment (willdollman) [reply dev/sg/internal/wolfi/base-image.go]: Nice

**Pull request merged**